kafka-rest/src/test/java/io/confluent/kafkarest/integration/AuthorizationErrorTest.java

```diff
@@ -50,12 +48,15 @@
 import org.apache.kafka.common.serialization.ByteArrayDeserializer;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.TestInfo;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 import scala.Option;
 
-/** This integration test uses AclAuthorizer class which is Zk specific. */
+// Until we fix KNET-16472, this test should be disabled.
+
+@Disabled
```

Comment:
[nit] you can put comment in the disabled tag
```suggestion
@Disabled("Until we fix KNET-16472, this test should be disabled")
```